targets/lenovo-x1-carbon.nix

```diff
@@ -268,6 +268,13 @@
             environment.etc.${getAuthKeysFilePathInEtc} = getAuthKeysSource {inherit pkgs;};
             services.openssh = sshAuthorizedKeysCommand;
 
+            # Enable TPM2 PKCS11 interface
+            security.tpm2.enable = true;
+            security.tpm2.pkcs11.enable = true;
+            security.tpm2.tctiEnvironment.enable = true;
+            environment.systemPackages = with pkgs; [tpm2-tools opensc];
```

Comment:
Probably good to move this to a module "modules/security/tpm.nix". Then just use config.ghaf.security.tpm2.enable or something similar to enable it here.